src/actions/update-wee-slack

```diff
@@ -0,0 +1,11 @@
+#!/usr/local/sbin/charm-env python3
+
+from lib_weechat import WeechatHelper
+from charmhelpers.core import hookenv
+
+helper = WeechatHelper()
+if helper.charm_config['enable-slack']:
+  helper.install_wee_slack()
+  hookenv.action_set({'outcome': 'success'})
+else:
+  hookenv.action_set({'outcome': 'failed'})
```

Comment:
What about a debug log here if the action is called but not run because of a config setting it would be hard to know why it failed. Or even put a message in the action output so people would know why it failed.